packages/cli/src/ui/hooks/useFolderTrust.test.ts

```diff
@@ -4,75 +4,161 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { renderHook, act } from '@testing-library/react';
 import { vi } from 'vitest';
+import { renderHook, act } from '@testing-library/react';
 import { useFolderTrust } from './useFolderTrust.js';
+import { type Config } from '@google/gemini-cli-core';
 import { LoadedSettings, SettingScope } from '../../config/settings.js';
 import { FolderTrustChoice } from '../components/FolderTrustDialog.js';
+import { loadTrustedFolders, TrustLevel } from '../../config/trustedFolders.js';
+import * as process from 'process';
+
+// Mock dependencies
+vi.mock('../../config/trustedFolders.js', () => ({
+  loadTrustedFolders: vi.fn(),
+  TrustLevel: {
```

Comment:
clean up how we are doing this. we shouldn't have to mock trustedFolders and we definitely shouldn't have to mock enums.